sonar-project.properties

```diff
@@ -1,3 +1,13 @@
-sonar.projectKey=centreon-dsm-20.10
-sonar.projectName=Centreon DSM 20.10
+# project
+sonar.projectKey={PROJECT_TITLE}
+sonar.projectName={PROJECT_NAME}
+sonar.projectVersion={PROJECT_VERSION}
 sonar.sources=.
+
+#exclusions
+sonar.exclusions=doc/**/*,\
```

Comment:
no issue if if it finished with `,\` ?

Reply:
In fact, it didn't trigger any warning or error 
`INFO: EXECUTION SUCCESS`

But I cleaned this line

Reply:
thanks !
Because there is the same thing here ;) https://github.com/centreon/centreon-awie/pull/45/files#diff-43ed9d31bea2a6d518d69836bcd1a8e6bd81bf4df96c4745792c220ca5aa549cR9